### PR TITLE
Move experimental features

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Move experimental features out of the 'experimental' directory, to their correct locations. Add the `experimental` tag (https://github.com/timescale/web-documentation/pull/220) and admonitions.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: api/page-index/page-index.js [hunk 1]
```diff
@@ -274,6 +282,10 @@ module.exports = [
               {
                 title: "interpolate",
                 href: "interpolate"
+              },
```
Comment (mfreed) [inline api/page-index/page-index.js:285]: Should we put `time_bucket_ng` right under `time_bucket` above?  (That feels right to me?)

If you are instead trying to put all experimental features at the end of the section -> then just one issue is I think this is incorrectly placed as a child of "gapfilling" in the hierarchy -> needs to be moved up one level.

**Review by mfreed — APPROVED** [review]

File: api/page-index/page-index.js (hunk 1 above)
Comment (Loquacity) [reply api/page-index/page-index.js]: Ah, good spot!

**Comment by Loquacity** [comment]
@jacobprall I'm not seeing the experimental tag? Is this because the test environment isn't using the latest code, or have I done something wrong?

image

**Review by jacobprall — COMMENTED** [review]
I've updated the front end to handle multiple tags (Community + Experimental), should be good to go!

**Pull request merged**